tests/common/dualtor/data_plane_utils.py

```diff
@@ -107,7 +108,9 @@ def validate_traffic_results(tor_IO, allowed_disruption, delay,
                             "Maximum allowed disruption: {}s"
                             .format(server_ip, longest_disruption, delay))
 
-        if total_duplications > allowed_disruption:
+        if allowed_duplication is None:
+            allowed_duplication = allowed_disruption
```

Comment:
Can you add comments for justify this assignment?

Reply:
Updated.

Reply:
Sorry, I meant to explain why allow_disruption value is a good fit for allow_duplicate? Why does it not affect the test results?

Reply:
Resolve per offline discussion.